code/modules/mob/living/carbon/human/physiology.dm

```diff
@@ -41,6 +41,9 @@
 	var/datum/armor/armor
 	///% of hunger rate taken per tick.
 	var/hunger_mod = 1
+	///bonus damage added to melee attacks. NOTE: this is additive rather than multiplicative, trying to multiply this will break things.
```

Comment:
```suggestion
	/// Bonus damage added to melee attacks. NOTE: this is additive rather than multiplicative, trying to multiply this will break things.
```